rs/nns/integration_tests/src/registry_get_chunk.rs

```diff
@@ -65,7 +65,7 @@ fn test_large_records() {
                 )
             ),
             version: new_version,
-            timestamp_seconds: 0,
+            timestamp_seconds: get_value_response.timestamp_seconds,
```

Comment:
This line doesn't test anything.  It could still be 0.  Probably there should be an assertion that it has some value that is in a reasonable range (i.e. greater than the timestamp the test started with)